.github/workflows/pr-title-linter.yml

```diff
@@ -16,15 +16,16 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
-
-      - name: Setup
-        id: config
-        uses: ./.github/actions/init
         with:
-          turbo-signature: ${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }}
-          turbo-team: ${{ vars.TURBO_TEAM }}
-          turbo-token: ${{ secrets.TURBO_TOKEN }}
-          playwright-enabled: true # Must be present to enable caching on branched workflows
+          fetch-depth: 1
+          show-progress: false
+          sparse-checkout: |
+            commitlint.config.ts
+            packages/*/package.json
+          sparse-checkout-cone-mode: false
 
       - name: Lint Pull Request Title
-        run: echo "${{ github.event.pull_request.title }}" | npx commitlint
+        run: |
+          npm init --scope=clerk --yes
+          npm i globby commitlint --audit=false --fund=false
```

Comment:
What's the usage of `globby` here?

Reply:
Globby is being used in the Commitlint config file itself.